internal/scorecard/plugins/plugin_runner.go

```diff
@@ -126,7 +126,7 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 	runtimeClient, _ = client.New(kubeconfig, client.Options{Scheme: scheme, Mapper: restMapper})
 
 	csv := &olmapiv1alpha1.ClusterServiceVersion{}
-	if pluginType == OLMIntegration {
+	if pluginType == OLMIntegration || config.OLMDeployed {
 		yamlSpec, err := ioutil.ReadFile(config.CSVManifest)
 		if err != nil {
```

Comment:
I was checking and we have not a mock test data with `olm-deployed: true`. So, this change will affect the both versions and then, we should add a mock for both scenarios with it. Could you please add it? 
See the existent tests/checks: https://github.com/operator-framework/operator-sdk/search?l=YAML&q=csv-path

Reply:
This is necessary but should go in a follow-up PR. We need to create a test environment that has the operator deployed on OLM before setting `olm-deployed: true`, which does not exist yet.

Reply:
In POV the test here is helpful to ensure the fix as well. So, for me make more sense the test to check/ensure this fix be done here. However,  l am ok with if the others are.